.github/workflows/test.yaml

```diff
@@ -86,42 +83,26 @@ jobs:
       - name: Install Nutils and dependencies
         id: install
         env:
-          _os: ${{ matrix.os }}
-          _matrix: ${{ matrix.matrix-backend }}
-          _numpy: ${{ matrix.numpy-version }}
+          _numpy_version: ${{ matrix.numpy-version }}
         run: |
-          _deps="coverage treelog stringly matplotlib pillow meshio"
-          case "$_os" in
-            windows-latest) _deps="$_deps psutil";;
-          esac
-          case "$_matrix" in
-            scipy) _deps="$_deps scipy";;
-            mkl) _deps="$_deps mkl";;
-          esac
-          case "$_numpy" in
-            latest) _deps="$_deps numpy Sphinx scipy";;
-            *) _deps="$_deps numpy==$_numpy";;
-          esac
           python -um pip install --upgrade wheel
-          python -um pip install --upgrade $_deps
-          # Install Nutils from `dist` dir created in job
-          # `build-python-package`.
+          python -um pip install --upgrade coverage treelog stringly matplotlib pillow meshio numpy$_numpy_version
+          # Install Nutils from `dist` dir created in job `build-python-package`.
           python -um pip install --no-index --find-links ./dist nutils
+      - name: Install Scipy
+        if: ${{ matrix.matrix-backend == 'scipy' }}
+        run: |
+          python -um pip install --upgrade scipy
       - name: Configure MKL
-        id: configure-mkl
         if: ${{ matrix.matrix-backend == 'mkl' }}
-        run: python -um devtools.gha.configure_mkl
+        run: |
+          python -um pip install --upgrade mkl
+          python -um devtools.gha.configure_mkl
       - name: Test
-        env:
-          NUTILS_DEBUG: all
         run: |
-          mkdir testenv
-          cp -r examples docs tests .coveragerc testenv
-          cd testenv
-          python -um coverage run -m unittest -bq
+          python -um coverage run -m unittest -bq tests/test_*.py
```

Comment:
Is listing `tests/test_*.py` necessary if we remove `examples/__init__.py`?

Reply:
Without this, python <= 3.6 stumbles over import dataclasses when it enters devtools during discovery...

Reply:
I don't like this much either

Reply:
What do you think of `python -m unittest discover -t . -s test`? Then you can also omit removing `examples/__init__.py`.

Reply:
done